bin/test_sequencer_solo

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash -e
+
+PROJECT_ID=$1
+shift 1
+
+SITE_PATH=sites/udmi_site_model
+DEVICE_ID=AHU-1
+
+RESULTS_OUT=/tmp/sequencer_solo.out
+PUBBER_OUT=/tmp/pubber.log
+SEQUENCER_OUT=/tmp/sequencer.log
+TEST_LIST=/tmp/solo_sequencer_tests.txt
+
+cat > $TEST_LIST <<EOF
```

Comment:
as per other comment, I think this should be parsed out of the in/out file, not as a separate list maintained here

Reply:
Changed to read the tests from the "golden file"